src/Portal/Portal.tsx

```diff
@@ -12,8 +12,8 @@ const portalRootRegistry: {[key: string]: Element} = {}
  * @param name The name of the container, to be used with the `containerName` prop on the Portal Component.
  * If name is not specified, registers the default portal root.
  */
-export function registerPortalRoot(root: Element, name = DEFAULT_PORTAL_CONTAINER_NAME): void {
-  portalRootRegistry[name] = root
+export function registerPortalRoot(root?: Element, name = DEFAULT_PORTAL_CONTAINER_NAME): void {
+  Object.assign(portalRootRegistry, {[name]: root})
```

Comment:
You can revert this and also remove the `registerPortalRoot(undefined)` call in your tests/stories.  This was made more robust and simpler to use in #1153.